### ISSUE TITLE
Control for random seed in `train_test_split ` in experimental branch

### ISSUE THREAD

**Issue by dsleo** [dataiku-research/cardinal#8]
Placeholder more than anything as it's not merged/nor PR-ready, just so we don't forget it. 
In the benchmarking script [here](https://github.com/dataiku/cardinAL/blob/experimental/examples/zhdanov_mnist.py#L82), we should control the `random_state`. So the permutation done before could be skipped with using the seed in place here.

**Comment by dsleo** [comment]
Way outdated, closing.
